Test/comp/Extern.dfy

```diff
@@ -16,4 +22,18 @@ module {:extern "Library"} Library {
     static method {:extern} CallMeInt(x: int) returns (y: int, z: int)
     static method {:extern} CallMeNative(x: MyInt, b: bool) returns (y: MyInt)
   }
+  
+  class AllDafny {
+    static function Seven(): int { 7 }
+    static method M() { print "AllDafny.M\n"; }
+  }
+  class {:extern} Mixed {
+    static function Seven(): int { 7 }
+    static method M() { print "Mixed.M\n"; }
+    static method {:extern} P()
+  }
+  class {:extern} AllExtern {
+    static function Seven(): int { 7 }
```

Comment:
Should this be `extern`?  (Given the name `AllExtern`)

Reply:
It's ghost, so it doesn't get compiled. (I suppose the name of the class could have been `AllExternOrGhost`.)